src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -457,6 +457,7 @@ public static string InvalidTokenProviderResponseValue(string invalidValueName)
         public const string CombinedUserAppCacheNotSupported = "Using a combined flat storage, like a file, to store both app and user tokens is not supported. Use a partitioned token cache (for ex. distributed cache like Redis) or separate files for app and user token caches. See https://aka.ms/msal-net-token-cache-serialization .";
         public const string JsonParseErrorMessage = "There was an error parsing the response from the token endpoint, see inner exception for details. Verify that your app is configured correctly. If this is a B2C app, one possible cause is acquiring a token for Microsoft Graph, which is not supported. See https://aka.ms/msal-net-up";
         public const string SetCiamAuthorityAtRequestLevelNotSupported = "Setting the CIAM authority (ex. \"{tenantName}.ciamlogin.com\") at the request level is not supported. The CIAM authority must be set during application creation";
-        public const string ClaimsChallenge = "The error returned contains a claims challenge. For additional info on how to handle claims related to multi-factor auth, conditional access and incremental consent, please see https://aka.ms/msal-conditional-access-claims. If you are using the On-behalf-of flow, please see https://aka.ms/msal-conditional-access-claims-obo";
+        public const string ClaimsChallenge = "The error returned contains a claims challenge. For additional info on how to handle claims related to multifactor authentication, conditional access, and incremental consent, see https://aka.ms/msal-conditional-access-claims. If you are using the On-Behalf-Of flow, see https://aka.ms/msal-conditional-access-claims-obo";

```

Comment:
```suggestion
        public const string ClaimsChallenge = "The returned error contains a claims challenge. For additional info on how to handle claims related to multifactor authentication, Conditional Access, and incremental consent, see https://aka.ms/msal-conditional-access-claims. If you are using the On-Behalf-Of flow, see https://aka.ms/msal-conditional-access-claims-obo.";
```

Reply:
There should be a space between a link and a period. `https://aka.ms/msal-conditional-access-claims-obo .` Otherwise, some pages (like the API reference) end up rendering the period as part of the link.

Reply:
Updated the comment @pmaytak - good catch.